libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1454,8 +1454,38 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 			if (equal)
```

Comment:
What happens if you remove the above part about mobile types? I don't see that part in the old code generator.

Reply:
Nothing actually. (i.e. no tests failing).